builtin/mainmenu/tab_credits.lua

```diff
@@ -26,25 +26,34 @@ local core_developers = {
 	"Auke Kok (sofar) <sofar@foo-projects.org>",
 	"Andrew Ward (rubenwardy) <rw@rubenwardy.com>",
 	"Krock/SmallJoker <mk939@ymail.com>",
-	"Lars Hofhansl <larsh@apache.org>",
+	"Lars Hofhansl <lhofhansl@yahoo.com>",
+	"Pierre-Yves Rollo <dev@pyrollo.com>",
+	"v-rob <robinsonvincent89@gmail.com>",
 }
 
 local active_contributors = {
-	"Hugues Ross [Formspecs]",
-	"Maksim (MoNTE48) [Android]",
-	"DS [Formspecs]",
-	"pyrollo [Formspecs: Hypertext]",
-	"v-rob [Formspecs]",
-	"Jordach [set_sky]",
-	"random-geek [Formspecs]",
-	"Wuzzy [Pathfinder, builtin, translations]",
-	"ANAND (ClobberXD) [Fixes, per-player FOV]",
-	"Warr1024 [Fixes]",
-	"Paul Ouellette (pauloue) [Fixes, Script API]",
-	"Jean-Patrick G (kilbith) <jeanpatrick.guerrero@gmail.com> [Audiovisuals]",
-	"HybridDog [Script API]",
-	"dcbrwn [Object shading]",
-	"srifqi [Fixes]",
+	-- TODO: add/keep areas of work here
+	"Wuzzy",
+	"Zughy",
+	"MoNTE48",
+	"numzero",
+	"Lars Müller",
+	"Desour",
+	"HybridDog",
+	"Hugues Ross",
+	"ClobberXD",
+	"Vincent Robinson",
+	"luk3yx",
+	"hecks",
+	"LoneWolfHT",
+	"Lejo",
+	"EvidenceB",
```

Comment:
not a person - but a company. Not sure how best to credit, this results in some inconsistency with pyrollo and kilbith